backend/coins/btc/account_test.go

```diff
@@ -193,3 +184,17 @@ func TestInsuredAccountAddresses(t *testing.T) {
 	require.Equal(t, *account2.GetUnusedReceiveAddresses()[0].ScriptType, signing.ScriptTypeP2WPKH)
 
 }
+
+func TestSignAddress(t *testing.T) {
+	account := mockAccount(t, nil)
+	require.NoError(t, account.Initialize())
+	// pt2r is not an available script type in the mocked account.
+	_, _, err := btc.SignBTCAddress(account, "Hello there", "p2tr")
+	require.NotNil(t, err)
+	address, signature, err := btc.SignBTCAddress(account, "Hello there", "p2wpkh")
+	require.Nil(t, err)
+	require.NotNil(t, address)
+	require.Equal(t, signature, base64.StdEncoding.EncodeToString([]byte("signature")))
```

Comment:
in Equal(), the first value is the expected and the 2nd the one tested for, so you should reverse the two.